src/pal/src/arch/i386/context.cpp

```diff
@@ -506,7 +574,7 @@ Return value :
 void CONTEXTToNativeContext(CONST CONTEXT *lpContext, native_context_t *native,
                             ULONG contextFlags)
 {
-    if (contextFlags != (CONTEXT_CONTROL | CONTEXT_INTEGER))
+    if ((contextFlags & (CONTEXT_CONTROL | CONTEXT_INTEGER)) != (CONTEXT_CONTROL | CONTEXT_INTEGER))
```

Comment:
No, the intent of the check is to verify that both the `CONTEXT_CONTROL` and `CONTEXT_INTEGER` are always specified and only the `CONTEXT_FLOATING_POINT` is optional.